tox.ini

```diff
@@ -78,7 +82,9 @@ deps =
     types-requests=={[format_deps]types-requests}
     types-setuptools=={[format_deps]types-setuptools}
     typing-extensions=={[format_deps]typing-extensions}
-commands = {envpython} -m mypy cloudinit/ tests/ tools/
+# TODO: add --check-untyped-defs to allow checking untyped function bodies
```

Comment:
This is done as of https://github.com/canonical/cloud-init/pull/5409 , correct?

Reply:
Yes!